tasmota/xdrv_17_rcswitch.ino

```diff
@@ -51,22 +51,21 @@ uint32_t rf_lasttime = 0;
 void RfReceiveCheck(void) {
   if (mySwitch.available()) {
 
-    unsigned long data = mySwitch.getReceivedValue();
+    unsigned long long data = mySwitch.getReceivedValue();
```

Comment:
Please prefer `uint64_t` as it makes its size explicit

Reply:
done